modules/administration-guide/partials/proc_configuring-bitbucket-servers.adoc

```diff
@@ -3,11 +3,8 @@
 [id="configuring_bitbucket_servers_{context}"]
 = Configuring Bitbucket servers
 
+To use the Bitbucket server as a project sources supplier:
 
-To make it possible to use the Bitbucket server as a project sources supplier, 
-Bitbucket server URL should be registered on {prod} using the `CHE_INTEGRATION_BITBUCKET_SERVER__ENDPOINTS` property.
-Value of the property must contain the hostname of the server to register.
-Examples on how to change configuration options using Helm or the Operator can be found here:
-
-* xref:installation-guide:advanced-configuration-options-for-the-che-server-component.adoc[]
+. Register the Bitbucket server URL on {prod-short} using the `CHE_INTEGRATION_BITBUCKET_SERVER__ENDPOINTS` property. The value of the property must contain the hostname of the server to register. For the configuration options changing examples, see: xref:installation-guide:advanced-configuration-options-for-the-che-server-component.adoc[]
```

Comment:
I'd suggest 'by using' instead of just 'using'. It decreases the ambiguity.

Reply:
New files could make it easier to follow:

```
. Register the Bitbucket server URL on {prod-short} using the `CHE_INTEGRATION_BITBUCKET_SERVER__ENDPOINTS` property.
+
The value of the property must contain the hostname of the server to register.
+
For the configuration options changing examples, see: xref:installation-guide:advanced-configuration-options-for-the-che-server-component.adoc[]
```

Reply:
Also, manuals of style tend to discourage breaking up a sentence with a colon after a verb. (Also, when a list is introduced with a colon, there has to be a complete sentence before the colon.) So if it were up to me, I would just drop the colon here:

`For the configuration options changing examples, see xref:installation-guide:advanced-configuration-options-for-the-che-server-component.adoc[]`